Assets/_Laz/Scripts/Laz/LazBoostBehaviour.cs

```diff
@@ -13,23 +14,7 @@ public class LazBoostBehaviour : MonoBehaviour, IBoost
         private float _elapsedBoostTime = 0;
         private bool _isBoostActivated = false;
 
-        public bool IsBoostActivated
-        {
-            get => _isBoostActivated;
-            set
-            {
-                _isBoostActivated = value;
-                ResetBoostTime();
-                if (_isBoostActivated)
-                {
-                    _lazMovement.ActivateBoost();
-                }
-                else
-                {
-                    _lazMovement.DeactivateBoost();
-                }
-            }
-        }
+        public bool IsBoostActivated => _isBoostActivated;
```

Comment:
Don't think putting side effect logic is the best way, since when other developers call this, they'll get side effects they don't know about.